Index: lib/spec/rails/context/controller.rb
===================================================================
--- lib/spec/rails/context/controller.rb	(revision 90)
+++ lib/spec/rails/context/controller.rb	(working copy)
@@ -44,7 +44,7 @@
             @template.stub!(:render)
             @template.stub!(:file_exists?).and_return(true)
             @template.stub!(:full_template_path)
-            @template.stub!(:render_file).and_return(true)
+            @template.stub!(:render_file).and_return("")
             @template.stub!(:render_partial)
           end
         end
